lib/puppet/provider/service/launchd.rb

```diff
@@ -196,6 +196,22 @@ def self.read_plist(path)
     Puppet::Util::Plist.read_plist_file(path)
   end
 
+  # Read overrides plist, retrying if necessary
+  def self.read_overrides
+    i = 1
+    overrides = nil
+    loop do
+      Puppet.debug("Reading overrides plist, attempt #{i}") if i > 1
```

Comment:
These should be of the form `_(<msg>) % key_hash`. We need `_` so that the messages are translated correctly. For example, here you could write:
```
Puppet.debug(_("Reading overrides plist, attempt %{i}") % {i: i})
```

Reply:
Done.